.github/workflows/core-network-services-deployment.yml

```diff
@@ -0,0 +1,59 @@
+---
+name: core-network-services-deployment
+
+on: 
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+    paths:
+      - 'terraform/environments/core-network-services/**'
+  workflow_dispatch:
+
+env:
+  AWS_ACCESS_KEY_ID:  ${{ secrets.AWS_ACCESS_KEY_ID }}
+  AWS_SECRET_ACCESS_KEY:  ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+  TF_IN_AUTOMATION: true
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  core-network-services-deployment-plan:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'pull_request'
+    steps:
+      - uses: actions/checkout@v2
+      - uses: hashicorp/setup-terraform@v1.3.2
+      - name: Run terraform plan in terraform/environments
+        run: |
+          scripts/terraform-init.sh terraform/environments/core-network-services
+          PLAN-DETAILS=`scripts/terraform-plan.sh terraform/environments/core-network-services`
+          scripts/terraform_plan_comment_update.sh ${PLAN-DETAILS}
```

Comment:
`./scripts/terraform_plan_comment_update.sh` might need to be `./scripts/update-pr-comments.sh` which is in this PR 👍🏼

Reply:
fixed